src/services/structures/structures.hooks.js

```diff
@@ -75,6 +79,31 @@ module.exports = {
           if (context.id.toString() !== context.params?.user?.entity?.oid.toString()) {
             throw new Forbidden('Vous n\'avez pas l\'autorisation');
           }
+          try {
+            const contact = context.data?.contact;
+            const structures = await context.app.service('structures').find({
```

Comment:
utilises plutôt `findOne` ça t'évitera de manipuler un tableau

Reply:
`structure` au singulier du coup

Reply:
au singulier pour le const , ok je rectif

Reply:
j'ai ajouter findOne mais j'ai cette error => error: context.app.service(...).findOne is not a function

Reply:
ha non c'est bon juste une petite erreur de synthaxe

Reply:
ha bha en faite j'ai toujours l'erreur